.gitignore

```diff
@@ -136,6 +136,7 @@ terraform.tfstate*
 localstack_providers_override.tf
 tf-apply.out
 tf-destroy.out
+errored.tfstate
```

Comment:
?

Reply:
some time when deploying locally, if the deployment gets error, it created a error state file